encoding/protobuf_test.go

```diff
@@ -22,11 +22,16 @@ func TestNewProtobuf(t *testing.T) {
 		},
 		{
 			desc:   "no method",
+			method: "Bar",
+			errMsg: "service \"Bar\" does not include a method named \"\", available methods:\n\tBar/Baz",
+		},
+		{
+			desc:   "missing method for service",
 			method: "Bar/baq",
-			errMsg: `service "Bar" does not include a method named "baq"`,
+			errMsg: "service \"Bar\" does not include a method named \"baq\", available methods:\n\tBar/Baz",
```

Comment:
nit: use backtick strings `service "Bar" does not include [...]`

same above

Reply:
I originally used the backticks but raw formatting with the tab and newline isn't much of an improvement:

```
                        method: "Bar",
-                       errMsg: "service \"Bar\" does not include a method named \"\", available methods:\n\tBar/Baz",
+                       errMsg: `service "Bar" does not include a method named "", available methods:
+       Bar/Baz`,
                },
```

Reply:
Makes sense, another option,
`fmt.Sprintf("service %q does not include a method named %q [...]", "Bar", [...])`

What you have now is fine too